gapis/api/vulkan/api/command_buffer_control.api

```diff
@@ -269,6 +269,14 @@ enum SemaphoreUpdate {
   @unused BufferCommands                  BufferCommands
   @unused ref!CommandBufferBegin          BeginInfo
   @unused ref!VulkanDebugMarkerInfo       DebugInfo
+
+  // Caching touched resources to help the state tracking performance
+  @hidden @unused @untracked @untrackedMap @nobox
```

Comment:
WE really need to start documenting these annotations somewhere. They might be getting slightly out of  hand.